components/script/dom/location.rs

```diff
@@ -78,7 +84,12 @@ impl LocationMethods for Location {
     // https://html.spec.whatwg.org/multipage/#dom-location-hash
     fn SetHash(&self, mut value: USVString) {
         if value.0.is_empty() {
-            value = USVString("#".to_owned());
```

Comment:
I think this change will make us fail tests? The spec has text specifically about how Location.hash's setter is supposed to behave when passed an empty string, and it's supposed to have # appended to it afterwards.